groups/utils.js

```diff
@@ -134,7 +134,7 @@ function getDiscordGroupIdsFromSearch(groups, multipleGroupSearch) {
     .map((group) => group.trim().toLowerCase());
   const matchGroups = groups.filter((group) =>
     searchGroups.some((searchGroup) =>
-      group.title.toLowerCase().startsWith(searchGroup),
```

Comment:
Why are you changing it to `includes`? It was the requirement to search for the prefix of the word(s)

Reply:
if we use only startsWith instead of includes the search will not return all relevant discord groups for a name, 

please see the image below                                                                                                                                                                                            
<details>
<summary>includes</summary>

![Screenshot 2024-10-15 141705](https://github.com/user-attachments/assets/ed5ac30a-2e1b-46af-9b98-f827c80516ae)
</details>

<details>
<summary>startsWith</summary>

![Screenshot 2024-10-15 141614](https://github.com/user-attachments/assets/6a60fee3-f2d5-485d-8695-f8c6cc279f8a)
</details>

Reply:
Which one will suit our requirement? 
Requriement: Prefix of the word(s)

Reply:
startsWith would be better if prefix is the requirement, will make changes

Reply:
changed it according to requirement i.e. startsWith